cypress/component/integration/Dropdown.cy.tsx

```diff
@@ -0,0 +1,289 @@
+                      size={32}
+                      className="bg-c-default rounded-full"
+                    />
+                  ),
+                  item: 'account 1 name',
+                  onClick: () => console.log('option 1'),
+                },
+                {
+                  icon: <MassaLogo size={32} />,
+                  item: 'account 2 name with a biiiiig content',
+                },
+                { icon: <MassaLogo size={32} />, item: 'account 3 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 4 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 5 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 6 name' },
+              ]}
+            />
+          </div>,
+        );
+      });
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-xs-dropdown');
+      });
+    });
+
+    describe('default md dropdown', () => {
+      beforeEach(() => {
+        cy.mount(
+          <div className="theme-dark">
+            <Dropdown
+              size={'md'}
+              options={[
+                {
+                  icon: (
+                    <MassaToken
+                      size={32}
+                      className="bg-c-default rounded-full"
+                    />
+                  ),
+                  item: 'account 1 name',
+                  onClick: () => console.log('option 1'),
+                },
+                {
+                  icon: <MassaLogo size={32} />,
+                  item: 'account 2 name with a biiiiig content',
+                },
+                { icon: <MassaLogo size={32} />, item: 'account 3 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 4 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 5 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 6 name' },
+              ]}
+            />
+            ,
+          </div>,
+        );
+      });
+
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-md-dropdown');
+      });
+    });
+
+    describe('default dropdown flow', () => {
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+
+        it('should toggle when clicked', () => {
+          cy.get('[data-testid="dropdown"]').should('exist');
+
+          for (let i = 1; i <= 6; i++) {
+            cy.get(`[data-testid="dropdown-option-${i}"]`).should('exist');
+          }
+
+          cy.get('[data-testid="dropdown-option-1"]').should('be.visible');
+          for (let i = 2; i <= 6; i++) {
+            cy.get(`[data-testid="dropdown-option-${i}"]`).should(
+              'not.be.visible',
+            );
+          }
```

Comment:
Except if you have a particular reason to have five elements in the dropdown I would recommend to create only 2 elements (less code also) and assert explicit here with no loops.